MeshLib/IO/FEFLOW/FEFLOWMeshInterface.cpp

```diff
@@ -6,7 +6,7 @@
  *              http://www.opengeosys.org/project/license
  */
 
-#include "FEFLOWInterface.h"
+#include "FEFLOWMeshInterface.h"
 
 #include <cctype>
 #include <QtXml>
```

Comment:
I am unsure what is included together with `QtXml`. The clang compiler at my system complains about "FEFLOWMeshInterface.cpp:377:34: error: member access into incomplete type 'QDomElement'".